bootstrap/config/kfctl_gcp_basic_auth_master.yaml

```diff
@@ -0,0 +1,228 @@
+# Please set project and email!
+apiVersion: kfdef.apps.kubeflow.org/v1alpha1
+kind: KfDef
+metadata:
+  creationTimestamp: null
+  name: myapp2
+  namespace: kubeflow
+spec:
+  repos:
+  - name: kubeflow
+    root: kubeflow-master
+    uri: https://github.com/kubeflow/kubeflow/archive/master.tar.gz
+  - name: manifests
+    root: master/
+    uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
+    # To get manifest at a PR:
+    #uri: https://github.com/kubeflow/manifests/archive/pull/235/head.tar.gz
+  appdir: /tmp/myapp2
+  componentParams:
```

Comment:
We don't want to use componentParams, packages, and components anymore.

We should be using KfDef.spec.Applications

The easiest way to convert them is to run kfctl and that will populate the app.yaml with the requisite KfDef.Spec.Applications

YOu could then copy and paste it here.

It might be sufficient to run kfctl generate

Reply:
done